concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-471	471	meet	1831	MEET	2
+Marrison-1967-909-472	472	mend	2243	REPAIR	4
+Marrison-1967-909-473	473	middle	1093	MIDDLE	2
+Marrison-1967-909-474	474	milk	635	MILK	2
+Marrison-1967-909-475	475	millet	931	MILLET	2
+Marrison-1967-909-476	476	mind	1562	MIND	2
+Marrison-1967-909-477	477	mist	249	FOG	2
+Marrison-1967-909-478	478	mistake	1293	MISTAKE	2
+Marrison-1967-909-479	479	mithan (gayal, bos fontalis)			
+Marrison-1967-909-480	480	money	1559	MONEY	2
+Marrison-1967-909-481	481	monkey (generic)	1350	MONKEY	4
+Marrison-1967-909-482	482	monkey (huluk)			4
+Marrison-1967-909-483	483	monkey (langur)			4
+Marrison-1967-909-484	484	monkey (macaque)			4
+Marrison-1967-909-485	485	moon	1313	MOON	2
+Marrison-1967-909-486	486	morning	1339	MORNING	2
+Marrison-1967-909-487	487	morung (men's dormitory)			
+Marrison-1967-909-488	488	mosquito	1509	MOSQUITO	2
+Marrison-1967-909-489	489	moth	2828	MOTH	2
+Marrison-1967-909-490	490	mother	1216	MOTHER	2
+Marrison-1967-909-491	491	mountain	639	MOUNTAIN	2
+Marrison-1967-909-492	492	mouse	1491	MOUSE	2
+Marrison-1967-909-493	493	mouth	674	MOUTH	2
+Marrison-1967-909-494	494	move	765	MOVE	2
+Marrison-1967-909-495	495	much	531	HOW MUCH	2
+Marrison-1967-909-496	496	mud	640	MUD	2
+Marrison-1967-909-497	497	must	971	MUST	2
+Marrison-1967-909-498	498	mustard			
+Marrison-1967-909-499	499	naked	590	NAKED	2
+Marrison-1967-909-500	500	name	1405	NAME	2
+Marrison-1967-909-501	501	narrow	1267	NARROW	2
+Marrison-1967-909-502	502	navel	1838	NAVEL	2
+Marrison-1967-909-503	503	near	1942	NEAR	2
+Marrison-1967-909-504	504	necessary	1030	NECESSARY	2
+Marrison-1967-909-505	505	neck	1333	NECK	2
+Marrison-1967-909-506	506	necklace	1961	NECKLACE	2
+Marrison-1967-909-507	507	needle	1382	NEEDLE (FOR SEWING)	2
+Marrison-1967-909-508	508	neighbor	1332	NEIGHBOUR	2
+Marrison-1967-909-509	509	nest	539	NEST	2
+Marrison-1967-909-510	510	net	2963	NET	2
+Marrison-1967-909-511	511	never	1656	NEVER	2
+Marrison-1967-909-512	512	new	1231	NEW	2
+Marrison-1967-909-513	513	news	2964	NEWS	2
+Marrison-1967-909-514	514	night	1233	NIGHT	2
+Marrison-1967-909-515	515	no	1269	NO	2
+Marrison-1967-909-516	516	noise	1182	NOISE	2
+Marrison-1967-909-517	517	noon	12	MIDDAY	2
+Marrison-1967-909-518	518	north	1598	NORTH	2
+Marrison-1967-909-519	519	nose	1221	NOSE	2
+Marrison-1967-909-520	520	not	1240	NOT	2
+Marrison-1967-909-521	521	now	1376	NOW	2
+Marrison-1967-909-522	522	oath	1712	OATH	2
+Marrison-1967-909-523	523	obey	1635	OBEY	2
+Marrison-1967-909-524	524	oil	1484	OIL (ORGANIC SUBSTANCE)	2
+Marrison-1967-909-525	525	old (of persons)	2112	OLD (AGED)	4
+Marrison-1967-909-526	526	old (of things)	2113	OLD (USED)	4
+Marrison-1967-909-527	527	old man	406	OLD MAN	2
+Marrison-1967-909-528	528	old woman	907	OLD WOMAN	2
+Marrison-1967-909-529	529	on	2379	UP OR ABOVE	4
+Marrison-1967-909-530	530	onion	2366	ONION	2
+Marrison-1967-909-531	531	only	1580	ONLY	2
+Marrison-1967-909-532	532	open	1156	OPEN	2
+Marrison-1967-909-533	533	orange	2627	ORANGE (FRUIT)	2
```

Comment:
is orange the color or the fruit, can you find this out? if not, maybe rather leave unlinked?

Reply:
the orthography file indicates fruits